src/http/vote-http.js

```diff
@@ -0,0 +1,29 @@
+import * as voteCore from '../core/vote-core';
+import {createJsonRoute, throwStatus} from '../util/express';
+import {assert} from '../validation';
+
+let putVote = createJsonRoute(function(req, res) {
+  const vote = assert(req.body, 'vote');
+  console.log(vote);
+
+  if (!req.client.id) {
+    throwStatus(403);
+  }
+
+  return voteCore.createOrUpdateVote({
+      vote: {
+        value: vote.value,
+        user_id: req.client.id,
+        feed_item_id: vote.feedItemId,
+      },
+      client: req.client,
+    })
+    .then(rowsInserted => rowsInserted)
```

Comment:
The return value is number of rows inserted? I think the documentation says it returns a feed item object